src/types/breadcrumbs.ts

```diff
@@ -11,6 +11,14 @@ export interface BreadcrumbItem {
 }
 
 export interface BreadcrumbProps {
+  /**
+   * Breadcrumb item list
```

Comment:
```suggestion
   * An array of breadcrumb items.
```

（I copied from the docs)